src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -56,9 +58,45 @@ public static String formatNote(String header, Note note) {
         return encloseTopAndBottom(formattedString);
     }
 
-    public static String formatTimetable(String header, Timetable timetable) {
-        String formattedString = "";
-        return formattedString;
+    /**
+     * Takes an array list of events and converts it to a formatted, non-indexed string for output.
+     *
+     * @param header Success message.
+     * @param year Year of timetable
+     * @param month Month of timetable
+     * @param timetable Events to be printed
+     * @return Formatted string of non-indexed events in timetable
+     */
+    public static String formatTimetable(String header, int year, int month,
+                                         HashMap<Month, HashMap<Integer, ArrayList<Event>>> timetable) {
+        String formattedString;
+        if (month != 0) {
+            formattedString = generatesHeader(header + String.format(" %d-%d", year, month));
+        } else {
+            formattedString = generatesHeader(header + " " + year);
+        }
+        ArrayList<String> unformattedEvents = Event.yearCalendarToString(timetable);
+        for (String event : unformattedEvents) {
+            formattedString = formattedString.concat(encloseRow(event));
+        }
```

Comment:
Possible to change the parameter such that it takes in an arrayList of events instead of HashMap<Month, HashMap<Integer, ArrayList<Event>>> timetable?

Reply:
While possible, I'd lose the information on the name of the month and at each event, I'll have to check the month and see if it's different at each step. I'll basically be shifting the processing of each event from Event to here. Is there any significant reason why? If not I feel there's no added benefit to doing so.

Reply:
If thats the case, I think can leave it. The only concern is that the data struct looks complicated haha.

Reply:
I'll see what I can do as I am already currently pushing in the year and month already. May be able to cut down the data struct shape